tests/kmstool-enclaves-cli/test-enclave/test.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -e
+
+# Receive payload from parent instance
+socat VSOCK-LISTEN:8001 OPEN:payload.json,creat,trunc
+
+ACCESS_KEY_ID=$(cat payload.json | jq -r '.AccessKeyId')
+SECRET_ACCESS_KEY=$(cat payload.json | jq -r '.SecretAccessKey')
+SESSION_TOKEN=$(cat payload.json | jq -r '.Token')
+KMS_KEY_ARN=$(cat payload.json | jq -r '.KMSKeyArn')
+CIPHERTEXT=$(cat payload.json | jq -r '.Ciphertext')
+
+# Set library path to app directory for libnsm.so
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/app
+
+# Run decrypte
+echo "======================================================================================"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                      Decrypt                                       |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "|                                                                                    |"
+echo "======================================================================================"
+
+/app/kmstool_enclave_cli decrypt \
+    --region us-east-1 \
+    --proxy-port 8000 \
+    --aws-access-key-id ${ACCESS_KEY_ID} \
+    --aws-secret-access-key ${SECRET_ACCESS_KEY} \
+    --aws-session-token ${SESSION_TOKEN} \
+    --ciphertext ${CIPHERTEXT}
```

Comment:
Would be more useful to have a status after each test rather than exit the script after the first failure.

Eg:
```
Decrypt: SUCCESS
GenKey: FAIL
...
===
Status: FAIL
```

Reply:
Added in 33a67da2e021764fd120505633f3ab81350a232a